Demo/Demo/Info.plist

```diff
@@ -3,7 +3,7 @@
 <plist version="1.0">
 <dict>
 	<key>CFBundleShortVersionString</key>
-	<string>1.5.0</string>
+	<string>2.0.0-beta1</string>
```

Comment:
I think this is handled in the release script, but fixed it before I saw the yml file. I think it's fine.